kolibri/auth/test/helpers.py

```diff
@@ -0,0 +1,70 @@
+"""
+Helper functions for use across the user/auth/permission-related tests.
+"""
+
+from ..models import FacilityUser, Facility, Classroom, LearnerGroup, FacilityDataset
+
+def create_dummy_facility_data(classroom_count=2, learnergroup_count=2):
+    """
+    Helper to bootstrap facility data for use in role/permission scenarios (collections, users, and roles).
+    This can be called multiple times to create parallel facilities/datasets in the same database, to test
+    cross-dataset permissions. Returns a dict containing objects and lists of objects that were created.
+
+    :param int classroom_count: (optional), the number of classrooms to create in the facility (defaults to 2)
+    :param int classroom_count: (optional), the number of learner groups to create in each classroom (defaults to 2)
+    :returns: total number of errors if strict is True, otherwise 0
```

Comment:
Sorry, I was hasty in writing this comment -- what I mean is that the docstring (particularly this line) doesn't seem to reflect what this function actually does.